lib/apiservers/portlayer/swagger.json

```diff
@@ -1411,6 +1411,137 @@
 				}
 			}
 		},
+		"/interaction/{id}/serial": {
+			"post": {
+				"description": "Add serial port",
```

Comment:
Let's avoid leaking the implementation detail (serial port) in the description. This call should remain viable when we switch to VMCI based backchannel.